.circleci/config.yml

```diff
@@ -26,13 +26,48 @@ jobs:
           name: code-coverage
           command: bash <(curl -s https://codecov.io/bash)
 
+      - run:
+          name: commitlint
+          command: |
+            echo "module.exports = {extends: ['@commitlint/config-conventional']}" > commitlint.config.js
```

Comment:
If we decide to use this tool, it's probably easier to check this file into git instead of echoing here

Reply:
It's done this way on purpose to remain agnostic of npm vs. non npm projects